deploy/admission_webhook.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: service-apis
+  labels:
+    name: service-apis
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: service-apis-admission
+webhooks:
+  - name: validate.networking.x-k8s.io
+    matchPolicy: Equivalent
+    rules:
+      - operations: [ "CREATE" , "UPDATE" ]
+        apiGroups: [ "networking.x-k8s.io" ]
+        apiVersions: [ "v1alpha1" ]
+        resources: [ "httproutes" ]
+    failurePolicy: Fail
+    sideEffects: None
+    admissionReviewVersions:
+      - v1
+    clientConfig:
+      service:
+        name: service-apis-admission
+        namespace: service-apis
+        path: "/validate"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: service-apis-webhook
+    version: 0.0.1
+  name: service-apis-admission
+  namespace: service-apis
+spec:
+  type: ClusterIP
+  ports:
+    - name: https-webhook
+      port: 443
+      targetPort: 8443
+  selector:
+    name: service-apis-admission
+---
+# TODO(cmluciano): Tone down logging level for webhook
+# once we are satified with it's stability.
+#
+# TODO(cmluciano): Swap image name to the k8s official namespace
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: service-apis-admission
+  namespace: service-apis
+  labels:
+    name: service-apis-admission
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: service-apis-admission
+  template:
+    metadata:
+      name: service-apis-admission
+      labels:
+        name: service-apis-admission
+    spec:
+      containers:
+        - name: webhook
+          image: cmluciano/service-apis-admission:latest
+          imagePullPolicy: Always
+          args:
+            - -alsologtostderr
```

Comment:
For some reason this is trying to log to a file too. I'm getting this error on startup:

```
log: exiting because of error: log: cannot create log: open /tmp/service-apis-webhook.service-apis-admission-64c576d476-q7xhn.webhook.log.INFO.20210203-060618.1: no such file or directory
```

Reply:
Ya this was due to me setting this as alsologtostderr, I needed it to be logtostderr